plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/TrinoHiveCatalog.java

```diff
@@ -650,4 +650,13 @@ public void renameMaterializedView(ConnectorSession session, SchemaTableName sou
         }
         return listNamespaces(session);
     }
+
+    @Override
+    public void updateColumnComment(ConnectorSession session, SchemaTableName schemaTableName, ColumnIdentity columnIdentity, Optional<String> comment)
+    {
+        metastore.commentColumn(schemaTableName.getSchemaName(), schemaTableName.getTableName(), columnIdentity.getName(), comment);
```

Comment:
Do we need to update the comment both in the metastore and via the Iceberg `updateColumnDoc`?

If we don't need the metastore's comment set we can move this code up to `IcebergMetadata` and make it catalog agnostic.

Reply:
I think we should remove both, as it leads to inconsistent state if the `icebergTable.updateSchema().updateColumnDoc` fails.

let's reconsider with a followup